### ISSUE TITLE
Update to 1.6.10

### ISSUE THREAD

**Issue by sentinelweb** [cfnz/muirwik#80]
I was forced to update my KMM project to 1.6.10. but now there are a few problems with my kotlinJS modules as there have been a few changes i guess. I had to update kotlin_styled and react libraries to:
ver_kotlin_react=17.0.2-pre.289-kotlin-1.6.10
ver_kotlin_styled=5.3.3-pre.289-kotlin-1.6.10  

but now i get some website errors :
```
Uncaught TypeError: RStatics is undefined
    <anonymous> webpack-internal:///./kotlin-dce-dev/muirwik-muirwik-components-js-legacy.js:99
    <anonymous> webpack-internal:///./kotlin-dce-dev/muirwik-muirwik-components-js-legacy.js:14553
    ...
```
i guess they are fairly easily fixable but i would probably need some hrlp / guidance to update it - would be prepaed to put in some time to update the library to 1.6.10 if i could get some help doing it. kotlin is movng fast!

**Comment by cfnz** [comment]
Version 5 branch was released some time ago, but just now merged into Master and included updates for 1.6.10, but note it has needed to switch to the legacy version.
Also check out possible retirement note on Muirwik in the readme.

**Comment by sentinelweb** [comment]
ok thanks for the update. and the good work...
